controller/instance_manager_controller.go

```diff
@@ -590,6 +590,10 @@ func (imc *InstanceManagerController) canDeleteInstanceManagerPDB(im *longhorn.I
 				hasHealthyReplicaOnAnotherNode = true
 				break
 			}
+			if r.Spec.HealthyAt == "" && r.Spec.FailedAt == "" && r.Spec.NodeID == imc.controllerID {
+				hasHealthyReplicaOnAnotherNode = true
```

Comment:
Please use an separate bool variable and add some comments for this case. e.g., `isUnusedReplicaOnCurrentNode`

Reply:
changed